src/onnxocr/onnx_paddleocr.py

```
@@ -1,10 +1,70 @@
+        normalized = str(model_name).lower()
+        size = next((name for name in PPOCRV6_MODEL_CONFIGS if name in normalized), None)
+    else:
+        size = None
+
+    if size not in PPOCRV6_MODEL_CONFIGS:
+        return None
+    return size
+
+
+def _build_ppocrv6_defaults(kwargs):
+    """为 PP-OCRv6 模型构建默认参数。如果不是 v6 模型则返回空字典，保证 v5 兼容。"""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)